### ISSUE TITLE
[Accessibility] Installed tab with a warning icon due to deprecation is not announced

### ISSUE THREAD

**Issue by nkolev92** [NuGet/Home#8518]
Install a deprecated package and go to the PM/Solution PM UI. 

Navigate to installed tab and notice that the deprecation warning is not announced. 

Note that if you have a number in the updates tab it is announced, so there is precedent for this.
